src/PowerShell/PowerShellManager.cs

```diff
@@ -106,6 +108,9 @@ internal void Initialize()
                 // Run the profile.ps1
                 InvokeProfile(FunctionLoader.FunctionAppProfilePath);
```

Comment:
It looks like we have a problem: profile invocation is not covered by OTel, correct? We don't have to solve it in this PR, but let's think about it. Eventually, we should cover it as well, as it often contains the most important diagnostics data.

Reply:
Agreed. Opened an issue from the module repo, even though the code change is likely here, just to track it together with all of the other OTel optimizations and discussions. Link: https://github.com/Azure/azure-functions-powershell-opentelemetry/issues/12